docs/src/appendix/cheatcodes/cheat_span.md

```diff
@@ -0,0 +1,48 @@
+# `CheatSpan`
+
+```rust
+enum CheatSpan {
+    Indefinite: (),
+    Calls: usize
+}
+```
+
+`CheatSpan` is an enum used to specify for how long the target should be cheated.
+- `Indefinite` applies the cheatcode for indefinitely, until the cheat is stopped manually (e.g. using `stop_warp`).
+- `Calls` applies the cheatcode for specified number of calls, after which the cheat is stopped. 
```

Comment:
I think we should call this behavior consistently. 
One time it appears as "cancelled" one time it's "stopped"